didx-xyz/yoma-mobile
# YOMA-662 - refactor(WorkExperience): enabling create work experience credentials

## Scope [YOMA-662](https://yomaworld.atlassian.net/browse/YOMA-662)

![](https://media.giphy.com/media/hOzfvZynn9AK4/giphy.gif)
## Work Done
- renamed Experience > Work Experience
- merged with Jobs module. 
- renamed UserJobs > UserWorkExperience.
- Fixed up the create Work Experience call.
- Cleaned up some styles on Create Work Experience form.  

## Steps to Test
- Open app > Work Experience. Press (+) button and create a work experience - should be created successfully.
